src/main/kotlin/org/akuleshov7/api/ApiJson.kt

```diff
@@ -0,0 +1,15 @@
+package org.akuleshov7.api
+
+import kotlinx.serialization.*
+
+@Serializable
+data class StargazersListJson(val list:List<StargazersJson>)
+
+@Serializable
+data class StargazersJson(val login: String, val id: String)
+
+@Serializable
+data class ReposJson(val full_name: String, val fork: Boolean)
```

Comment:
```suggestion
data class ReposJson(val fullName: String, val isFork: Boolean)
```
Where is your diktat code style check? )

Reply:
@petertr, ah, boy, it is the json from github API unfortunately, not able to change :((

Reply:
Well, you could write `@SerialName("fork") val isFork: Boolean`, but I'm not sure if following code style is worth introducing more complexity )